graylog2-web-interface/src/components/common/GridsterContainer.jsx

```diff
@@ -63,14 +82,14 @@ const GridsterContainer = React.createClass({
   },
   _onPositionsChange() {
     const positions = this.state.grid.serialize().map((position) => {
-      return {id: position.id, col: position.col, row: position.row, width: position.size_x, height: position.size_y};
+      return { id: position.id, col: position.col, row: position.row, width: position.size_x, height: position.size_y };
     });
     this.props.onPositionsChange(positions);
   },
 
   render() {
     const children = (this.state.grid && React.Children.map(this.props.children, (child) => {
-      const position = this.props.positions[child.props.id] || {row: 0, col: 0, width: 1, height: 1};
+      const position = this.props.positions[child.props.id];
```

Comment:
Why are you doing this? What happens when there is no position for the widget (for some reason)?